gnomad_qc/v5/sample_qc/create_sample_qc_metadata_ht.py

```diff
@@ -0,0 +1,634 @@
+        "duplicate_or_twin": DUPLICATE_OR_TWINS,
+        "parent_child": PARENT_CHILD,
+        "sibling": SIBLINGS,
+    }
+    relationships = relationship_ht[ht.key]
+    relatedness_filters_ht = ht.annotate(
+        relatedness_filters=hl.struct(
+            **{
+                rel: get_relationship_filter_expr(
+                    hard_filtered_expr,
+                    hl.is_defined(related_samples_to_drop(release=False).ht()[ht.key]),
+                    relationships.relationships,
+                    rel_val,
+                )
+                for rel, rel_val in rel_dict.items()
+            }
+        ),
+        release_relatedness_filters=hl.struct(
+            **{
+                rel: get_relationship_filter_expr(
+                    hard_filtered_expr | outlier_filtered_expr,
+                    hl.is_defined(related_samples_to_drop(release=True).ht()[ht.key]),
+                    relationships.relationships_high_quality,
+                    rel_val,
+                )
+                for rel, rel_val in rel_dict.items()
+            }
+        ),
+    )
+
+    return relatedness_filters_ht
+
+
+def main(args):
+    """Merge the output of all sample QC modules into a single Table."""
+    hl.init(
+        spark_conf={"spark.memory.offHeap.enabled": "false"},
+        log="/home/jupyter/workspaces/gnomadproduction/create_sample_meta.log",
+        tmp_dir=f"gs://{WORKSPACE_BUCKET}/tmp/4_day",
+    )
+
+    try:  # Add AoU hard filters to the meta Table.
+        meta_ht = prepare_meta_with_hard_filters(
+            meta_ht=project_meta.ht(),
+            samples_to_exclude=get_samples_to_exclude(),
+            aou_hard_filters_ht=hard_filtered_samples.ht(),
+        )
+        # Restructure metadata fields.
+        meta_ht = restructure_meta_fields(meta_ht=meta_ht)
+
+        # Annotate genetic ancestry inference.
+        meta_ht = annotate_genetic_ancestry(
+            meta_ht=meta_ht,
+            v4_meta_ht=v4_meta("4.0", "genomes").ht(),
+            aou_gen_anc_ht=get_gen_anc_ht(projection_only=True).ht(),
+        )
+        # Add sample filter annotations.
+        meta_ht = add_sample_filter_annotations(
+            meta_ht=meta_ht, outlier_filters_ht=finalized_outlier_filtering().ht()
+        )
+
+        # Add relatedness inference and filters to the metadata Table.
+        meta_ht = add_relatedness_inference(
+            meta_ht=meta_ht,
+            relatedness_ht=relatedness().ht(),
+            outlier_filters_ht=finalized_outlier_filtering().ht(),
+        )
+
+        logger.info("\n\nAnnotating release field...")
+        meta_ht = meta_ht.annotate(
+            project_meta=meta_ht.project_meta.annotate(
+                releasable=hl.if_else(
+                    meta_ht.project_meta.project == "aou",
+                    True,
+                    meta_ht.project_meta.releasable,
+                )
+            )
+        )
+
+        meta_ht = meta_ht.annotate(
+            release=(
```

Comment:
Because release was hanging around from the project meta and not restructured it gets put right after s in the HT. This is very minor but maybe drop and reannotate so its closer to the last rows of the HT like previous releases?